OpenOversight/app/models.py

```diff
@@ -571,16 +586,14 @@ def reset_password(self, token, new_password):
         return True
 
     def generate_email_change_token(self, new_email, expiration=3600):
-        s = Serializer(current_app.config["SECRET_KEY"], expiration)
-        return s.dumps({"change_email": self.id, "new_email": new_email}).decode(
-            "utf-8"
-        )
+        key = current_app.config["SECRET_KEY"]
```

Comment:
Do you think we could move this step into `_jwt_encode` and `_jwt_decode`? That way the key wouldn't have to be pulled every time we call either of those functions.

Reply:
Yep, good point!